moped-editor/src/components/GridTable/GridTable.js

```diff
@@ -493,20 +470,10 @@ const GridTable = ({
                                         )
                                       ) : isAlphanumeric(column) ? (
                                         <>
-                                          {query.config.columns[
-                                            column
-                                          ].hasOwnProperty("badge")
-                                            ? buildStatusBadge(
-                                                row[column],
-                                                row[
-                                                  query.config.columns[column]
-                                                    .badge
-                                                ]
-                                              )
-                                            : query.getFormattedValue(
-                                                column,
-                                                row[column]
-                                              )}
```

Comment:
The Grid table had support for rendering a status badge as a hold over from when we used it in the project list, but we've since moved on to material table. We only use the GridTable component on the Staff page.